client/src/cli.rs

```diff
@@ -370,6 +369,135 @@ pub async fn start_cli_client(attach: SocketAddr) -> Result<()> {
     Ok(())
 }
 
+/**
+ * Process a CLI command from the client, we are the server side.
+ */
+async fn process_cli_command(
+    guest: &Arc<Guest>,
+    fw: &mut FramedWrite<tokio::net::tcp::OwnedWriteHalf, CliEncoder>,
+    cmd: protocol::CliMessage,
+    ri: &mut RegionInfo,
+    wc_filled: &mut bool,
+    vi: Option<PathBuf>,
```

Comment:
This should probably be "verify_input" instead, I had to search for what it meant :)

Reply:
Fixed in 263d810